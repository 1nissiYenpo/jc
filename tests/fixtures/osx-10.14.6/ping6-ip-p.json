{"source_ip": "::1", "destination_ip": "::1", "data_bytes": 56, "pattern": "0xff", "destination": "::1", "packets_transmitted": 3, "packets_received": 3, "packet_loss_percent": 0.0, "duplicates": 0, "round_trip_ms_min": 0.077, "round_trip_ms_avg": 0.124, "round_trip_ms_max": 0.156, "round_trip_ms_stddev": 0.034, "responses": [{"type": "reply", "bytes": 16, "response_ip": "::1", "icmp_seq": 0, "ttl": 64, "time_ms": 0.077, "duplicate": false}, {"type": "reply", "bytes": 16, "response_ip": "::1", "icmp_seq": 1, "ttl": 64, "time_ms": 0.156, "duplicate": false}, {"type": "reply", "bytes": 16, "response_ip": "::1", "icmp_seq": 2, "ttl": 64, "time_ms": 0.139, "duplicate": false}]}
