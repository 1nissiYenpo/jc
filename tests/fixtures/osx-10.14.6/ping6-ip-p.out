PATTERN: 0xff
PING6(56=40+8+8 bytes) ::1 --> ::1
16 bytes from ::1, icmp_seq=0 hlim=64 time=0.077 ms
16 bytes from ::1, icmp_seq=1 hlim=64 time=0.156 ms
16 bytes from ::1, icmp_seq=2 hlim=64 time=0.139 ms

--- ::1 ping6 statistics ---
3 packets transmitted, 3 packets received, 0.0% packet loss
round-trip min/avg/max/std-dev = 0.077/0.124/0.156/0.034 ms
